modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -229,16 +229,26 @@ definition {
 		}
 	}
 
-	@description = "This is a test for LPS-173731. Verify that is possible to export a searched MDF"
+	@description = "This is a test for LPS-174679.Verify that is possible to export filtered MDF’s"
```

Comment:
Hi @juliosilveira77, Could you please add a blank space after the `...LPS-174679.`? Just to separate the ticket number from the test description. Thanks!